.codeclimate.yml

```diff
@@ -2,7 +2,8 @@
 version: "2"
 plugins:
   brakeman:
-    enabled: true
+    enabled: false
+    # until they fix rails 6 handling
```

Comment:
is this fixed now?

Reply:
I can re-enable it to check but signs point to no <https://docs.codeclimate.com/docs/brakeman>

Reply:
It doesn't look like this will be fixed anytime soon but brakeman does apparently have github actions options we could try instead if we want to keep it, it seems to be a codeclimate problem not a brakeman problem.